guides/common/modules/proc_recovering-from-disaster-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,17 @@
+[id="recovering-from-disaster-with-active-and-passive-server-with-external-storage"]
+= Recovering from disaster with active and passive server with external storage
+
+If your active {ProjectServer} fails, detach it from the network-attached storage (NAS) and make sure your passive server can access the data stored on the NAS.
+This turns the passive server into your new active server.
+
+.Procedure
+. Verify that the failed active server is powered off or fully detached from the NAS.
+This ensures that the failed server cannot keep writing to the NAS, which helps prevent further data corruption on the connected databases.
```

Comment:
`further data corruption` might not be entirely precise -- if it wasn't writing anything before failing, then there was no data corruption, in which case `further` would be slightly misleading (worst case scenario, a very literally-minded reader might take some unnecessary actions on their supposedly corrupted database). I've no good ideas how to phrase it though, perhaps something like this?

  `which helps prevent data corruption on connected databases from starting or worsening`
  or
  `which helps prevent connected databases from experiencing the start or worsening of data corruption`

Reply:
How about just dropping the `which [...]` part? Simply saying that we don't want the failed server writing to the storage should be enough.